### PR TITLE
Show/Hide Invite People button

### PR DESCRIPTION
**Pull request by utkarshmarwaha**
Allows the user to show/hide "Invite people" button and "You are along" text on the conference page
Use key 'invite.enabled' as true or false to hide or show invite people button

### CODE REVIEW

File: react/features/conference/components/native/LonelyMeetingExperience.js [hunk 1]
```diff
@@ -65,22 +71,28 @@ class LonelyMeetingExperience extends PureComponent<Props> {
...
+        } = this.props;
 
         if (!_isLonelyMeeting) {
             return null;
         }
 
         return (
             <View style = { styles.lonelyMeetingContainer }>
-                <Text
+                { _inviteEnabled && (<Text
```
Comment (saghul) [inline react/features/conference/components/native/LonelyMeetingExperience.js:88]: You could just return null if this is set.

File: react/features/conference/components/native/LonelyMeetingExperience.js [hunk 2]
```diff
@@ -129,7 +141,8 @@ function _mapStateToProps(state): $Shape<Props> {
     return {
         _isInviteFunctionsDiabled: disableInviteFunctions,
         _isLonelyMeeting: getParticipantCount(state) === 1,
-        _styles: ColorSchemeRegistry.get(state, 'Conference')
+        _styles: ColorSchemeRegistry.get(state, 'Conference'),
+        _inviteEnabled: getFeatureFlag(state, INVITE_ENABLED, true)
```
Comment (saghul) [inline react/features/conference/components/native/LonelyMeetingExperience.js:145]: please sort this alphabetically

File: react/features/conference/components/native/LonelyMeetingExperience.js [hunk 3]
```diff
@@ -41,7 +41,13 @@ type Props = {
...
      * Function to be used to translate i18n labels.
      */
-    t: Function
+    t: Function,
+
+    /**
+     * Whether the invite people button is visible.
+     * Default: Visible (true)
+     */
+    _inviteEnabled: boolean,
```
Comment (saghul) [inline react/features/conference/components/native/LonelyMeetingExperience.js:50]: please sort this alphabetically

**Comment by Mohamed-Ramadan** [comment]
@utkarshmarwaha Hi, how can i use your custom repo after adding this feature ?

I tried to use your repo link into MatrixSDK.podspec, but got that error while trying to do pod install
`[!] The `MatrixSDK` pod failed to validate due to 1 error:
    - ERROR | [iOS] attributes: Unable to validate `dependencies` (Illformed requirement `{:git=>"https://github.com/utkarshmarwaha/jitsi-meet.git"}`).
`

**Comment by utkarshmarwaha** [comment]
@Mohamed-Ramadan 
To use my forked library, you can add it as a dependency in package.json file with version as git+https://github.com/utkarshmarwaha/jitsi-meet.git#COMMIT_HASH

Replace the COMMIT_HASH with the one you want to use.

**Comment by saghul** [comment]
Oops, I think I implemented this together with other feature flags. When invite.enabled is set to false we won’t show it.

**Pull request closed without merging**